src/main/java/org/jenkinsci/plugins/stashNotifier/StashNotifier.java

```diff
@@ -23,6 +23,7 @@
 import com.cloudbees.plugins.credentials.common.StandardListBoxModel;
 import com.cloudbees.plugins.credentials.common.UsernamePasswordCredentials;
 import com.google.inject.Injector;
+import edu.umd.cs.findbugs.annotations.NonNull;
```

Comment:
Note to myself. This is how it's done within Jenkins itself

https://github.com/jenkinsci/jenkins/pull/4604/files